src/github/webhook.rs

```diff
@@ -204,24 +204,31 @@ fn parse_issue_comment_event(body: &[u8]) -> anyhow::Result<Option<BorsEvent>> {
 fn parse_pull_request_events(body: &[u8]) -> anyhow::Result<Option<BorsEvent>> {
     let payload: WebhookPullRequestEvent = serde_json::from_slice(body)?;
     let repository_name = parse_repository_name(&payload.repository)?;
-    if payload.action == PullRequestEventAction::Edited {
-        let Some(changes) = payload.changes else {
-            return Err(anyhow::anyhow!(
-                "Edited pull request event should have `changes` field"
-            ));
-        };
-        Ok(Some(BorsEvent::Repository(
-            BorsRepositoryEvent::PullRequestEdited(PullRequestEdited {
-                repository: repository_name,
+    match payload.action {
```

Comment:
I forgot about it before, but we have some tests in this file for actually parsing the GH webhooks for the individual webhooks. Could you please add new tests for PR edited + PR synchronize? It's also nice to have some actual reference data for the webhook's HTTP contents.